test/display_test.py

```diff
@@ -783,6 +784,73 @@ def test_set_gamma_ramp(self):
         pygame.display.quit()
 
 
+class FullscreenToggleTests(unittest.TestCase):
+    __tags__ = ["interactive"]
+
+    screen = None
+    font = None
+    isfullscreen = False
+
+    WIDTH = 800
+    HEIGHT = 600
+
+    def setUp(self):
+        if self.screen is None:
```

Comment:
I think init should run regardless of the current `self.screen` state (because this function will overwrite it anyways)... also because `tearDown` does not set this to `None`, the `setUp` function will run only once, instead of before every test like we want it to

Reply:
Fair point, made those changes for you now